doc/options.md

```diff
@@ -895,6 +896,47 @@ a
 }
 ```
 
+## rule-delimiter
+
+Number of line breaks after a ruleset.
+
+Acceptable values:
+
+* `{Number}` — number of newlines;
```

Comment:
Hmm okay. Should this even be supported then? I'm not sure if spaces makes much sense in this case...